src/status_im/keycard/delete_key.cljs

```diff
@@ -74,6 +39,6 @@
                                                  :puk         []
                                                  :status      nil
                                                  :error-label nil
-                                                 :on-verified :keycard/remove-key-with-unpair})}
+                                                 :on-verified (if keep-keys-on-keycard? :keycard/unpair-and-delete :keycard/remove-key-with-unpair)})}
```

Comment:
Could you please make it three lines expression? It won't fit 80 chars anyway but this way it is too long

Reply:
Done. Thanks for the review!